tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    ceph_version = " "
+    for data in (response.text).split("\n"):
+        if "ceph-version" in data:
```

Comment:
In `RHCEPH-5.1.yaml` you will find lot of ceph-version, also when 5.1 z2 is planned in this yaml ceph-version gets updated based on build, So it's good to cross verify it from GAed ceph [article](https://access.redhat.com/solutions/2045583) In that case we are also validating the Doc which most customers and users look for it.
Please cross validate the version from GAed article?

Reply:
Thanks @sunilangadi2 
In "RHCEPH-5.1.yaml" we are targeting only latest key.

 As a part of "cross validate the version from GAed article" we are facing below issue.
1. GAed article is dynamic elements(Using Javascript while rendering the website).
2. As a result we are unable to locate versions in article.
3. We have try using python libraries like python request and Beautiful soup.
4. GAed article is dynamic in nature we may encounter failure in future run.

Reply:
@MohitBis ok then can we use this[ release.yaml](http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/release.yaml) 
@manasagowri can we add `ceph-version` in this? So that it helps easy to validate this test case.

Reply:
@sunilangadi2 In this case we will have to maintain the yaml file on a regular interval. Are we good with that ?

Reply:
@vdas-redhat Yes in Live testing we can have the script to get updated this release.yaml file.

Reply:
@sunilangadi2 
As discussed added "http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/release.yaml" link for verification.

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1WOS0B